pkg/rancher-components/src/components/Banner/Banner.vue

```diff
@@ -5,30 +5,46 @@ import { stringify } from '@shell/utils/error';
 
 export default Vue.extend({
   props: {
+    /**
+     * A color class that represents the color of the banner.
+     * @values primary, secondary, success, warning, error, info
```

Comment:
JSDoc doesn't support `@values` but I liked having a way to communicate what values a prop might accept.